### PR TITLE
[expo-cli][xdl] add deprecation notice + send cli version to www

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: packages/expo-cli/src/exp.js [hunk 1]
```diff
@@ -411,6 +416,22 @@ Run \`npm install -g ${packageJSON.name}\` to get the latest version`),
       )
     );
   }
+
+  if (deprecated) {
+    log.nestedWarn(
+      boxen(
+        chalk.red(
+          `This version of expo-cli is not supported anymore. 
+It's highly recomended to update to newest version.
```
Comment (dsokal) [inline packages/expo-cli/src/exp.js]:
```suggestion
It's highly recommended to update to the newest version.
```

File: packages/xdl/src/Api.js [hunk 2]
```diff
@@ -154,7 +157,7 @@ async function _downloadAsync(url, outputPath, progressFunction, retryFunction)
     response.data
       .on('data', chunk => {
         downloadProgress += chunk.length;
-        const roundedProgress = Math.floor((downloadProgress / totalDownloadSize) * 100);
+        const roundedProgress = Math.floor(downloadProgress / totalDownloadSize * 100);
```
Comment (dsokal) [inline packages/xdl/src/Api.js]: 😂

File: packages/expo-cli/src/exp.js [hunk 3]
```diff
@@ -411,6 +416,22 @@ Run \`npm install -g ${packageJSON.name}\` to get the latest version`),
...
+
+Api endpoint that were used up until now might not exist anymore or
```
Comment (dsokal) [inline packages/expo-cli/src/exp.js]: Sth is wrong with this sentence 🤔

File: packages/xdl/src/Api.js (hunk 2 above)
Comment (wkozyra95) [reply packages/xdl/src/Api.js]: this was linter change

**Pull request merged**